data/projects.js

```diff
@@ -182,4 +182,50 @@ exists."
 		"initiatives" : ["GSoC"],
 		"tags" : ["Core", "Bears"]
 	}
+	{
+		"name" : "Debugging and profiling bears",
+		"desc" : "If someone writes a bear he/she needs to test it and comes\
+likely to the step where debugging is needed. In this project the participant \
+shall develop a system to easily step into the analysis code (likely using the\
+ Pdb module already integrated into Python) and step out as soon as the \
+ analysis finished.\
+\
+On this way it would be useful retrieving profiling information while \
+debugging/running.",
+		"requirements" : [
+			"At least one patch to the coala core should be accepted and merged.",
+			"The student should be familiar with ongoing technical cEPs.",
+			"The proposal needs to contain a clear concept.",
+			"Familiarity with pdb is a plus."
+		],
+		"difficulty" : "high",
+		"issues" : [
+			"https://github.com/coala/coala/issues/565",
+			"https://github.com/coala/coala/issues/1101"
+		],
+		"mentors" : [
+			"sils",
+			"Makman2",
+			"Udayan12167"
+		],
+		"milestones" : {
+			"GSoC 2017 Coding Starts" : [
+				"The applicant has made a cEP of how the profiling will be \
+implemented",
+				"The applicant is familiar with pdb.",
+				"The applicant is familiar with core coala codebase."
+			],
+			"GSoC 2017 Midterm" : [
+				"The execution jumping directly into Bear code.",
+				"Easy settings investigation of Bears.",
+			],
+			"GSoC 2017 Final" : [
+				"Add profile option to CLI.",
```

Comment:
not "Add something" but rather "Something is done" or "something is documented, tested and merged" or so